### DISCUSSION TITLE
Useful Links

### DISCUSSION THREAD

**Discussion by ghost** [astarub/campus_app#62] [Ideas]
I think the Useful Link fit better within the Guide Page, what do you think? Why is it under more?

**Comment by henry-herrmann** [comment]
@ThetaEdra The guide page is pretty packed already so it would add even more content to that specific page. Makes sense to have it separated from commonly asked questions.

**Reply by ghost** [reply]
I thought of the Term Guide that it would Guide me somewhere like those links guide to useful information. i did not expect to see them under the section "more". I thought it was a little weird but if it is intended for now thats good to know.

**Comment by C4s4r** [comment]
As I already said in https://github.com/astarub/campus_app/discussions/70#discussioncomment-4458183, the guide page will be changed and therefore get a new name.
The useful links are currently placed in the _more_ page to have a central page to access all external services, that we don't develop ourself. Even if we implement them natively in the app, it makes sense to group them like they are now.
